pavelib/assets.py

```diff
@@ -597,6 +607,19 @@ def process_npm_assets():
     """
     Process vendor libraries installed via NPM.
     """
+    def copy_vendor_library(library, skip_if_missing=False):
```

Comment:
Agreed, and IMO this already feels ugly even before we get to NPM 3. I don't want to tackle this here, so I'll make a new ticket for it.